imtest/imtest.c

```diff
@@ -637,13 +637,13 @@ static long bio_dump_cb(BIO * bio, int cmd, const char *argp, int argi,
         return (ret);
 
     if (cmd == (BIO_CB_READ | BIO_CB_RETURN)) {
-        printf("read from %08lX [%08lX] (%d bytes => %ld (0x%lX))\n",
+        printf("read from %0lX [%0lX] (%d bytes => %ld (0x%lX))\n",
```

Comment:
(same comment as for the similar change in imap/tls.c, and the other one a few lines down from here)